test/unit/twophase_test.lua

```diff
@@ -0,0 +1,79 @@
+local t = require('luatest')
+local g = t.group()
+
+local helpers = require('test.helper')
+local fio = require('fio')
+
+g.server = t.Server:new({
+    command = helpers.entrypoint('srv_empty'),
```

Comment:
```suggestion
    command = helpers.entrypoint('srv_basic'),
```

Reply:
The test is failed if I change `srv_empty`  to `srv_basic`:
```
.rocks/bin/luatest -v unit.twophase
Started on Tue Jan 11 21:10:55 2022
    unit.twophase.test_timeouts ... (5.112s) error
...orov/cartridge/.rocks/share/tarantool/luatest/server.lua:191: Connection refused
=========================================================

Tests with errors:
------------------

1) unit.twophase.test_timeouts
...orov/cartridge/.rocks/share/tarantool/luatest/server.lua:191: Connection refused
stack traceback:
		/Users/andrei.sidorov/cartridge/test/unit/twophase_test.lua:23: in function 'retrying'
	/Users/andrei.sidorov/cartridge/test/unit/twophase_test.lua:23: in function </Users/andrei.sidorov/cartridge/test/unit/twophase_test.lua:21>
	...
	[C]: in function 'xpcall'
```